src/content/blog/eliminating-array-duplicates.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Eliminating Duplicates in Arrays
+author: Rob Maltese
+pubDatetime: 2022-02-13T18:34:31Z
+postSlug: eliminating-duplicates-in-arrays
+featured: false
+draft: false
+tags:
+  - swift
+  - swiftui
+  - array
+description:
+  While working with arrays, you may be tasked in making sure said array does not have any duplicates. We'll cover a few various methods to complete that within this article.
+---
+
+Arrays are an ordered random access collection in which contains elements of a defined data type. These types can be anything from an `Int` to a defined model. An array is something that you will see very often within Swift projects when organizing your apps data. The downfall to an array is that these items are not unique and the collection may contain duplicates.
```

Comment:
> in which

Reads weird.. maybe just use "which"

> Arrays are an ordered random access collection which contains elements of a defined data type.

Reply:
> are not unique and the collection may contain duplicates

Repetitive wording.. Maybe say something about order?

Reply:
Removed `in` and kept the `which`. Removed the second sentence as I mention order in the section about Set.

Changes reflected [on this commit](https://github.com/0xOpenBytes/Site/pull/25/commits/c2f4afc147871467b04eeb4e21cec637ed9946e4)